src/macros.hpp

```diff
@@ -114,7 +114,7 @@ namespace pinocchio
 #endif
 
 #ifndef PINOCCHIO_ENABLE_TEMPLATE_INSTANTIATION
-  #if __cplusplus >= 201103L
+  #if PINOCCHIO_WITH_CXX11_SUPPORT
```

Comment:
What is the reason for wrapping a single line macro inside another macro ?

I think the first version is more readable. You don't have to go to the definition of `PINOCCHIO_WITH_CXX11_SUPPORT` to know what it does.

Reply:
Yes, I do agree. But I also have in mind that the second version is readable for any users.